scripts/update_zynthian.sh

```diff
@@ -53,9 +53,12 @@ rm -f $REBOOT_FLAGFILE
 
 echo "Updating zynthian-sys ..."
 cd $ZYNTHIAN_SYS_DIR
+# Move users from master to stable - master is deprecated
+if [ `git rev-parse --abbrev-ref HEAD` == master ]; then git checkout stable; fi
```

Comment:
By doing on this way, you have to update twice. Also, it's better to put this "one-time" stuff on the recipes.update.